recipe/meta.yaml

```diff
@@ -154,16 +153,48 @@ test:
     - pyomo.solvers.plugins.solvers
     - pyomo.util
     - pyomo.version
-
+  source_files:
+  # Tests are spread across many files throughout the source code.
```

Comment:
`pytest --pyargs pyomo` should be able to find all the tests.

Reply:
Is there a way to specify files to ignore during collection when using `--pyargs`? `test_piecewise_linear_kernel.py` triggers an error during collection otherwise:

```
_ ERROR collecting solvers/tests/piecewise_linear/test_piecewise_linear_kernel.py _
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.8/site-packages/pyomo/solvers/tests/piecewise_linear/test_piecewise_linear_kernel.py:147: in <module>
    assignTests(PiecewiseLinearKernelTest, problems)
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.8/site-packages/pyomo/solvers/tests/piecewise_linear/test_piecewise_linear_kernel.py:131: in assignTests
    with open(
E   FileNotFoundError: [Errno 2] No such file or directory: '$PREFIX/lib/python3.8/site-packages/pyomo/solvers/tests/piecewise_linear/kernel_baselines/convex_var_baseline_results.json'
```

Reply:
@ViridianMelody have a look at this commit https://github.com/AnacondaRecipes/alibi-detect-feedstock/pull/1/commits/4e14153c892b00cac0dd57117da92586233826e7